frontend/src/components/Showtimes/Showtimes.component.vue

```diff
@@ -0,0 +1,349 @@
+    confirmDelete: false,
+    deletingShowtime: 0,
+    editingShowtime: new Showtime(),
+    theaterId: 0,
+    createDialogBool: false,
+    showtime: new Showtime(),
+    modal: false,
+    time: null,
+    showtimes: [],
+    showShowtimes: false,
+    movieName: '',
+    editDialog: false
+  }),
+  computed: {
+    filteredMovies() {
+      if (!_.isEmpty(this.search)) {
+        return this.movies.filter((i) => {
+          return i.title.toLowerCase().includes(_.trim(this.search.toLowerCase()));
+        });
+      }
+      return this.movies;
+    },
+    ...mapGetters([
+      'activeUser'
+    ])
+  },
+  mounted() {
+    this.getTheaterAndMovies();
+  },
+  methods: {
+    createDialog(movie) {
+      this.createDialogBool = true;
+      this.showtime.movie = movie.id;
+    },
+    create() {
+      this.showtime.time = this.time;
+      this.createDialogBool = false;
+      ShowtimeController.create(this.showtime)
+        .then((response) => {
+          this.$alert.success('Successfully added!');
+        })
+        .catch((response) => {
+          this.$alert.error('Error occurred.');
+        });
+    },
+    opetDialog() {
+      this.showDialog = true;
+    },
+    clear() {
+      this.showtime = new Showtime();
+    },
+    getTheaterAndMovies() {
+      this.loading = true;
+      TheaterController.getTheater(this.activeUser.id)
+        .then((response) => {
+          this.theaterId = response.data.id;
+          this.getMovies();
+          this.loading = false;
+        })
+        .catch((response) => {
+          this.loading = false;
+          this.$alert.error('Error occurred.');
+        });
+    },
+    getMovies(id = this.theaterId) {
+      this.loading = true;
+      TheaterController.getMovies(id)
+        .then((response) => {
+          this.movies = response.data;
+          this.loading = false;
+        })
+        .catch((response) => {
+          this.loading = false;
+          this.$alert.error('Error occurred.');
+        });
+    },
+    show(movieId) {
+      MovieController.getShowtimes(movieId)
+        .then((response) => {
+          this.showtimes = response.data;
+          this.showShowtimes = true;
+        })
+        .catch((resposne) => {
+          this.$alert.error('Error occurred.');
+        });
+    },
+    deleteButton(id) {
+      this.confirmDelete = true;
+      this.deletingShowtime = id;
+    },
+    remove() {
+      this.confirmDelete = false;
+      ShowtimeController.destroy(this.deletingShowtime)
+        .then((response) => {
+          this.$alert.success('Successfully deleted!');
+        })
+        .catch((response) => {
+          this.$alert.error('Error occurred.');
+        });
+    },
+    editButton(showId) {
+      ShowtimeController.retrieve(showId).then((response) => {
+        this.editingShowtime = new Showtime(response.data);
+      }).catch(() => {
+        this.$alert.error('Error occurred!');
+      });
+      this.editDialog = true;
```

Comment:
mozda ovo u then, da se ne prikaze ako ne uspe da dovuce podatke sa be

Reply:
ne kontam ovo... mi imamo vec `showtimes` u data. jel to lista modela? ako jeste ne moras uopste da dovlacis opet sa ovim `.retrieve` nego samo menjas ovaj iz liste i onda sibas update